src/Event/Http/HttpResponse.php

```diff
@@ -53,11 +58,17 @@ public function toApiGatewayFormat(bool $multiHeaders = false): array
 
         // This is the format required by the AWS_PROXY lambda integration
         // See https://stackoverflow.com/questions/43708017/aws-lambda-api-gateway-error-malformed-lambda-proxy-response
-        return [
+        $response = [
             'isBase64Encoded' => $base64Encoding,
             'statusCode' => $this->statusCode,
             $headersKey => $headers,
             'body' => $base64Encoding ? base64_encode($this->body) : $this->body,
         ];
+
+        if (isset($cookies)) {
+            $response['cookies'] = $cookies;
+        }
```

Comment:
At this point I'm afraid the `cookies` key might be added to the v1.0 format too, and the `Set-Cookies` header will be removed (from v1.0 responses) 🤔